static/src/stylesheets/module/commercial/_labs-capi.scss

```diff
@@ -397,7 +397,7 @@
     }
 
     .fc-item__standfirst {
-        color: $neutral-1;
+        color: #757575;
```

Comment:
Magic hexes! (boo)

This is the color we use for the paid-for badge. Ideally, we will overhaul the labs colours, consolidate them, and contain them to _vars.scss as a true part of the [palette set](http://guardian.github.io/pasteup-palette/demo/).

This would be a good thing to work on next quarter.